buildspec.yml

```diff
@@ -15,7 +17,7 @@ phases:
       - echo Building the Docker image...
       - chmod +x ./update_data.sh
       - echo Getting latest GeoLite2 ASN DB
-      - wget https://geolite.maxmind.com/download/geoip/database/GeoLite2-ASN.tar.gz
+      - wget https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-ASN&license_key=`$SUA_GEOLITE_ACCESS_KEY`&suffix=tar.gz
```

Comment:
The environment variable should be accessed using just variable expansion, not sub-shell expansion. By using backticks, your asking the shell to call an executable whose name is stored in the variable, e.g.:

```bash
$ MY_STRING="pwd"

# With backticks, STDOUT from the pwd command is substituted
$ echo BEFORE-`$MY_STRING`-AFTER
BEFORE-/home/.../dev/SpeedUpAmerica-AFTER

# FYI, $() does the same and can be nested
$ echo BEFORE-$($MY_STRING)-AFTER
BEFORE-/home/.../dev/SpeedUpAmerica-AFTER

# Without backticks, the raw string is substituted
$ echo BEFORE-$MY_STRING-AFTER
BEFORE-pwd-AFTER
```

You can still delineate the variable name, just use curly braces (handy when the variable is followed by characters that are also valid in variable names). This is the style I'd recommend:
```bash
$ echo BEFORE${MY_STRING}AFTER
BEFOREpwdAFTER
```